translation/message_format.rst

```diff
@@ -45,12 +45,12 @@ The basic usage of the MessageFormat allows you to use placeholders (called
 
     .. code-block:: yaml
 
-        # translations/messages.en.yaml
+        # translations/messages+intl-icu.en.yaml
         say_hello: 'Hello {name}!'
 
     .. code-block:: xml
 
-        <!-- translations/messages.en.xlf -->
+        <!-- translations/messages+intl-icu.en.xlf -->
```

Comment:
In `trabslation/debug.rst` we are using `.xliff` as extension:
```rst
.. configuration-block::

    .. code-block:: xml

        <!-- translations/messages.fr.xliff -->
        <?xml version="1.0"?>
        <xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
            <file source-language="en" datatype="plaintext" original="file.ext">
                <body>
                    <trans-unit id="1">
                        <source>Symfony is great</source>
                        <target>J'aime Symfony</target>
                    </trans-unit>
                </body>
            </file>
        </xliff>
```

But in general we a re using `.xlf`, which one should we communicate in the docs?

Reply:
I think `xlf` is the recommended one. We're using it in the Symfony Demo too: https://github.com/symfony/demo/tree/master/translations

Reply:
I will change the other occurrences then in a separate PR